NumSet/NumSet/Writer/SetWriter.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Assingment4.Writer
+{
+    public class SetWriter : IDisposable
+    {
```

Comment:
What you have set up here is a folder, not a nested namespace.

```suggestion
namespace Assingment4
{
    namespace NestedAssignment4 // This is your nested namespace
    {
        public class SetWriter : IDisposable
        {
```